RetailSuiteWarehousing/Integration/01_ConfigureWarehouseGroups.md

```diff
@@ -0,0 +1,186 @@
+[!!Configure the warehouses](./02_ConfigureWarehouses.md)
+[!!Check the posting history](../Operation/03_CheckPostingHistory.md)
+[!!User interface Warehouse](../UserInterface/03a_Warehouse.md)
+
+# Configure the warehouse groups
+
+A warehouse group is a higher-level classification of storage facilities according to customer-defined criteria, for example location or type of product. Warehouse groups are physically and conceptually separated from each other and therefore one delivery note must be issued for each warehouse group. A warehouse group can contain an unlimited number of warehouses.  
+
+Warehouses can be created, edited, and deleted. Bear in mind, though, that you cannot delete a warehouse group where postings have been made in the past, as all posting information is kept in the system. For detailed information, see [Check the posting history](../Operation/03_CheckPostingHistory.md).
+
+
+
+## Create a warehouse group
+
+Create a warehouse group to which you can assign any number of specific warehouses, for example a sales warehouse, a returns warehouse, or a storage area for blocked stock.
```

Comment:
Hier ganz andere Beispiele als in Satz oben drüber. Welche Klassifizierung ist empfehlenswert, was sind Vor- und Nachteile?